runtime/doc/starting.txt

```diff
@@ -1338,12 +1338,12 @@ DATA DIRECTORY (DEFAULT) ~
 Note: Throughout the user manual these defaults are used as placeholders, e.g.
 "~/.config" is understood to mean "$XDG_CONFIG_HOME or ~/.config".
 
-LOG FILE					*$NVIM_LOG_FILE*
+LOG FILE					*$NVIM_LOG_FILE* *E5010*
 Besides 'debug' and 'verbose', Nvim keeps a general log file for internal
 debugging, plugins and RPC clients. >
 	:echo $NVIM_LOG_FILE
-Usually the file is ~/.cache/nvim/log unless that path is inaccessible
-or if $NVIM_LOG_FILE was set before |startup|.
+By default, the file is located at $XDG_CACHE_HOME/nvim/log unless that path 
```

Comment:
this is valid only on linux, may be best to refer to it as stdpath('cache')

Reply:
Done

Reply:
Also added a note that logs are stored at the standard location now (along with data)